data-prepper-plugins/anomaly-detector-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/anomalydetector/modes/RandomCutForestModeConfig.java

```diff
@@ -25,22 +26,27 @@ public class RandomCutForestModeConfig {
 
     public static final String VERSION_1_0 = "1.0";
 
+    @JsonPropertyDescription("The algorithm version number.")
```

Comment:
Might be nice to mention the default of all of these in the JsonPropertyDescription